person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = 1
```

Comment:
- [ ] [OPTIONAL] if every person will have an id of 1 it will be hard deleting a person in the future, you might want to consider making this a random number.

Reply:
Well, I thought of that, but all the same, the computer can generate two same numbers with that method. I will do it that way anyway hoping to see how we solve the problem permanently as the project progresses. Thank you.